charts/sonarqube-dce/values.yaml

```diff
@@ -145,7 +145,7 @@ ApplicationNodes:
 
   # Pod Distribution Budget for search nodes
```

Comment:
Typo in comment, should mention APP instead of Search.

Reply:
good catch!